cdap-docs/developers-manual/source/building-blocks/core.rst

```diff
@@ -32,7 +32,7 @@ seamlessly deploy it in a different environment—such as your distributed stagi
 .. rubric:: Data and Applications Combined
 
 With your data virtualized in CDAP as *Streams* and *Datasets*, you are able to process
-that data in realtime or in batch using Programs (*Flows, MapReduce Job, Spark Job,
+that data in realtime or in batch using Programs (*Flows, MapReduce program, Spark program,
```

Comment:
Some this list is inconsistent w.r.t. plural vs. singular. Either flows, programs, workflows, or flow, program, workflow?